float64 RadarDx
float64 RadarDy
float64 RadarVx
float64 RadarVy
float64 RadarTimestamp

